ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -65,13 +65,76 @@ module('Acceptance | sync | overview', function (hooks) {
     await click(ts.overview.optInConfirm);
   });
 
-  test('it should show adapter error if call to activated-features fails', async function (assert) {
-    assert.expect(2);
-    this.server.get('/sys/activation-flags', () => {
-      assert.ok(true, 'Request on initial load to check if secrets-sync is activated');
-      return AdapterError.create();
+  module('enterprise with namespaces', function (hooks) {
+    hooks.beforeEach(async function () {
+      await runCmd(`write sys/namespaces/admin -f`, false);
+      await authPage.loginNs('admin');
+      await runCmd(`write sys/namespaces/foo -f`, false);
+      await authPage.loginNs('admin/foo');
+    });
+    test('it should make activation-flag requests to correct namespace', async function (assert) {
+      assert.expect(6);
+      this.server.get('/sys/activation-flags', (_, req) => {
+        assert.deepEqual(req.requestHeaders, {}, 'Request is unauthenticated and in root namespace');
+        return {
+          data: {
+            activated: [''],
+            unactivated: ['secrets-sync'],
+          },
+        };
+      });
+      this.server.post('/sys/activation-flags/secrets-sync/activate', (_, req) => {
+        assert.strictEqual(
```

Comment:
def don't wanna block the release for this, but seems like we could test the specifics of the API call within a route unit test instead of an acceptance test. something to talk about later!